### DISCUSSION TITLE
Font Weight firefox/chrome

### DISCUSSION THREAD

**Discussion by mila76** [quasarframework/quasar#10052] [General - Components / Directives / etc] [answered]
Take a look ad the buttons for examples...
Chrome:
![btn_chrome](https://user-images.githubusercontent.com/378500/125293275-8b939e80-e323-11eb-99e6-04bc682ce432.PNG)
Firefox:
![btn_firefox](https://user-images.githubusercontent.com/378500/125292961-41122200-e323-11eb-9c5c-f777b1ca6137.PNG)
In chrome is clearly bold and firefox is like normal fonts

I know there are difference (for how font handling work on different browser and OS) and for sure can't be fixed for now but my question is....
Is the rendering in firefox or in chrome the more "correct" or to be more precise the "wanted one"?? 

Thanks for anyone feedback on this matter

**Comment by pdanpdan** [comment, marked as answer]
From experience, Firefox is the one that renders wrong, but you can check by playing with font-weight and checking what is displayed.

**Reply by mila76** [reply]
Yeah yeah i cant make a css fix for this  (For now i put q-btn font-weight in firefox to be 600 and the result is more similar) but i dont know what the quasarar design teams use for a reference
Is the chrome the desired? Or maybe neither of bot because they use linux and ther is different
